hooks/pre/13-dns_check.rb

```diff
@@ -0,0 +1,21 @@
+# Check to verify forward/reverse dns matches hostname
+
+if app_value(:disable_system_checks)
+  logger.warn 'Skipping system checks.'
+else
+  hostname = `$(hostname -f)`
```

Comment:
I think this is redundant: backticks execute shell and ```$()``` is a bash subshell so you can remove the subshell.